scripts/ci

```diff
@@ -21,19 +21,20 @@ function format_check() {
     scripts/check_copyright
 }
 
-function contracts_check() {
+function check_contracts() {
     pushd zeth_contracts
     npm config set python python2.7
     npm install
     npm run check
     popd
 }
 
-function pyclient_check() {
+function check_pyclient() {
```

Comment:
While this is being changed, let's change `pyclient` to `client`.

Reply:
Indeed. Switched to that